Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -380,6 +390,89 @@
         <UnlocalizedDescription>furn_fission_reactor_desc</UnlocalizedDescription>
     </Furniture>
 
+    <Furniture type="water_generator">
+        <Name>Water Generator</Name>
+        <TypeTag>Generator</TypeTag>
+        <MovementCost>2</MovementCost>
+        <PathfindingModifier>0</PathfindingModifier>
+        <PathfindingWeight>2</PathfindingWeight>
+        <Width>1</Width>
+        <Height>1</Height>
+
+
+        <OrderAction type="Build">
+            <Job time="1" />
+            <Inventory type="Steel Plate" amount="2" />
+        </OrderAction>
+                
+        <OrderAction type="Deconstruct">
+            <Job time="1" />
+            <Inventory type="Steel Plate" amount="1" />
+        </OrderAction>
+<!--
+        <Component type="Workshop">
+            <ParameterDefinitions>
+                <CurrentProcessingTime name="cur_processing_time" />
+                <MaxProcessingTime name="max_processing_time" />
+                <IsProcessing name="cur_processed_inv" />
+                <CurrentProductionChainName name="cur_production_chain" />
+            </ParameterDefinitions>
+            <ProductionChain name="Power generation" processingTime="60">
+                <Input>
+                    <Item objectType="Power Cell" amount="1" slotPosX="0" slotPosY="0" hasHopper="false" />
+                </Input>
+            </ProductionChain>
+        </Component>
+    -->    
+        <Component type="FluidConnection">
```

Comment:
this can be implemented in workshop component, but would require some additional flags.. and maybe some changes on how requirements (power, fluids, ... ) are handled (updated)